resources/docs/editing-chirps.md

```diff
@@ -383,7 +383,7 @@ class ChirpPolicy
     public function update(User $user, Chirp $chirp)
     {
         //
-        return $user->id === $chirp->user_id; // [tl! remove:-1,1 add]
+        return $chirp->user()->is($user); // [tl! remove:-1,1 add]
```

Comment:
Will it load `user` relation from `chirp` and only after that the `id` will be get from `chirp`'s user?

Reply:
This is calling the `is` method on the `BelongsTo` relation object, which doesn't need to load anything to perform the comparison. If it was `$chirp->user->is($user)` then it would fetch the user.